servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -2064,6 +2064,10 @@ void RenderForwardMobile::_fill_render_list(RenderListType p_render_list, const
 			surf = surf->next;
 		}
 	}
+
+	if (p_render_list == RENDER_LIST_OPAQUE && lightmap_captures_used) {
+		RD::get_singleton()->buffer_update(scene_state.lightmap_capture_buffer, 0, sizeof(LightmapCaptureData) * lightmap_captures_used, scene_state.lightmap_captures);
+	}
```

Comment:
Note that despite this being done in the opaque render list, it works on dynamic objects that have transparent materials too.